### DISCUSSION TITLE
How to Exit Vim 🤣

### DISCUSSION THREAD

**Discussion by bennypowers** [rmagatti/auto-session#113] [Q&A] [answered]
Ok, Ok bear with me for a minute, I know I'm a meme, it's fine

Before I installed auto-session, I'd quit vim the way G-d intended, by mashing <kbd>ZZ</kbd> until all the buffers and tabs were closed.

But now that I've installed auto-session, I'd prefer to 'just quit' without first closing open buffers and tabs, etc.

So, what's the command to just exit the process without mucking about with the session?

**Comment by bennypowers** [comment, marked as answer]
`:quitall` is the answer 😉

**Reply by rmagatti** [reply]
`:qa` for short 😉

**Reply by bennypowers** [reply]
Thanks :D
